src/chart/line/LineView.js

```diff
@@ -521,6 +522,10 @@ export default ChartView.extend({
                     // Null data
                     return;
                 }
+                // fix #11360: don't draw symbol if clip is true and symbol is outside the visible area
+                if (seriesModel.get('clip', true) && !this._clipShapeForSymbol.contain(pt[0], pt[1])) {
```

Comment:
`this._clipShapeForSymbol` will be `undefined` if `seriesModel.get('clip', true)` is `false` or coordinateSystem didn't have `getArea` method. Check code https://github.com/apache/incubator-echarts/blob/master/src/chart/line/LineView.js#L357

So is it better to be written as 
```js
if (this. _clipShapeForSymbol && !this._clipShapeForSymbol.contain(pt[0], pt[1])) {
```

Reply:
OK, thx.